ddp-workspace/projects/ddp-lms/src/assets/i18n/en.json

```diff
@@ -489,5 +489,88 @@
+      "Advisor8": "Nikky Hughes",
+      "Advisor9": "Miyoung Kim",
+      "Advisor10": "Bella Pesina",
+      "Advisor11": "Yemi Getachew",
+      "ClinicalChampions": "Clinical Champions",
+      "Champion1": "Megan Anderson",
+      "Champion2": "Brian Crompton",
+      "Champion3": "Suzanne George",
+      "Champion4": "Candace Haddox",
+      "Champion5": "Priscilla Merriam",
+      "Champion6": "Chandrajit Raut",
+      "Champion7": "Michael Nathenson",
+      "Champion8": "Jay Oza",
+      "ChampionBio1": "Boston Children’s Hospital\nHarvard Medical School \n",
+      "ChampionBio2": "Dana-Farber Cancer Institute\nBoston Children’s Hospital\nHarvard Medical School\nBroad Institute of MIT and Harvard\n",
+      "ChampionBio3": "Dana-Farber Cancer Institute\nHarvard Medical School \n",
+      "ChampionBio4": "Dana-Farber Cancer Institute\n",
+      "ChampionBio5": "Dana-Farber Cancer Institute\nHarvard Medical School \n",
+      "ChampionBio6": "Brigham and Women’s Hospital\nDana-Farber Cancer Institute\nHarvard Medical School\n",
+      "ChampionBio7": "Brigham and Women’s Hospital\nDana-Farber Cancer Institute\nHarvard Medical School\n",
+      "ChampionBio8": "Dana-Farber Cancer Institute\nHarvard Medical School \n",
+      "LearnMoreHeadline": "Learn more about the Leiomyosarcoma Project",
+      "VisitFaq": "Visit FAQ"
+    }
+  },
+  "FYP": {
+    "FYP": {
+      "headerText": "For Your Physician",
+      "Purpose": "The purpose of this page is to provide details about the Leiomyosarcoma Project to your physician and care team so they can learn more about the project. You can show them this website, or print out a PDF below with the same information to discuss at your next appointment",
+      "downloadTitle": "Download printable PDF to share with your care team",
+      "paragraphTitle1": "The Leiomyosarcoma Project (LMSproject) ",
+      "paragraphTextPart1": "is part of Count Me In, a nonprofit organization\nthat brings together patients and researchers as partners to accelerate discoveries in cancer\nresearch. Count Me In is stewarded by the Broad Institute of MIT and Harvard, a leading\nnonprofit biomedical research institution; the Dana-Farber Cancer Institute, a leading cancer\nhospital; and Emerson Collective, a California-based social change organization. Count Me In's\nLeiomyosarcoma (LMS) Project is supported by a grant from the National Cancer Institute's\nCancer Moonshot program and is part of the",
+      "paragraphTextPart2": "Participant Engagement and Cancer Genome\nSequencing (PE-CGS) Network",
```

Comment:
I think this should be a clickable link. Please ask the ticket author to provide the URL.

![image](https://user-images.githubusercontent.com/96420397/169008124-a4996ced-9c85-46b9-ac9e-5bbd50a644ed.png)

Reply:
theu, answered me that it is not klickable form